src/test/resources/en/ParagraphLinksTest.txt

```diff
@@ -0,0 +1,5 @@
+This [[document]] has been created to test links in paragraphs.
```

Comment:
Ok the tests are good, but i think we should steal some test cases from the previous pull requests and see if the methods here work as well.